mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
Can you use GLib's [internationalization support](https://developer.gnome.org/glib/stable/glib-I18N.html) for these definitions?